setup.py

```diff
@@ -62,13 +63,7 @@ def read_description():
         'Intended Audience :: End Users/Desktop',
         'Intended Audience :: Manufacturing',
         'Intended Audience :: Science/Research',
-        'Topic :: Scientific/Engineering :: Information Analysis',
-        'Topic :: Education',
-        'Topic :: Scientific/Engineering',
-        'Topic :: Scientific/Engineering :: Artificial Intelligence',
-        'Topic :: Scientific/Engineering :: Human Machine Interfaces',
-        'Topic :: Scientific/Engineering :: Mathematics',
-        'Topic :: Scientific/Engineering :: Physics',
+        'Topic :: Scientific/Security :: Metadata leakage',
```

Comment:
This classifier is not a standard one. It is recommended that you choose from classifiers listed [here](https://pypi.org/classifiers/).

Reply:
My suggestion:

1. `Topic :: Security`
2. `Topic :: Utilities`

Reply:
done.